python/ray/tests/spark/test_GPU.py

```diff
@@ -55,9 +65,28 @@ def test_gpu_allocation(self):
                 ray.init()
                 worker_res_list = self.get_ray_worker_resources_list()
                 assert len(worker_res_list) == num_worker_nodes
+
+                num_ray_task_slots = self.max_spark_tasks // (
+                    num_gpus_worker_node // self.num_gpus_per_spark_task
+                )
+                (
+                    mem_per_worker,
+                    object_store_mem_per_worker,
+                    _,
+                ) = _calc_mem_per_ray_worker_node(
+                    num_task_slots=num_ray_task_slots,
+                    physical_mem_bytes=_RAY_ON_SPARK_WORKER_PHYSICAL_MEMORY_BYTES,
+                    shared_mem_bytes=_RAY_ON_SPARK_WORKER_SHARED_MEMORY_BYTES,
+                    configured_object_store_bytes=None,
+                )
+
```

Comment:
Is this related to autoscaling?

Reply:
ditto: https://github.com/ray-project/ray/pull/38215#discussion_r1338339922